sdk/eventhub/azure-eventhub/TROUBLESHOOTING.md

```diff
@@ -82,33 +95,57 @@ An `AuthenticationError` means that the provided credentials do not allow for th
 
 This error can occur when an intercepting proxy is used.  We recommend testing in your hosting environment with the proxy disabled to verify.
 
-### Socket exhaustion errors
+### Managing Clients
+
+Applications should manage how they open and close the eventhub client.  It is essential to be aware that your application is responsible for calling `close()` when it is finished using a client or to use the `with` statement for clients so that they are automatically closed after the flow execution leaves that block. This make sures that the socket connection to the server is closed correctly. If the client is not closed this can leave AMQP connections open, sockets open and prevent other clean up activities from running.
+
+### Adding components to the connection string does not work
```

Comment:
```suggestion
### Cannot add components to the connection string
```

Reply:
This section should be moved below the IoT one to match the order of the list at the top.